src/libs/getReceiptsUploadFolderPath/index.android.ts

```diff
@@ -0,0 +1,7 @@
+import RNFetchBlob from 'react-native-blob-util';
```

Comment:
They are not really the same because the path is different:
- Android: `${RNFetchBlob.fs.dirs.DownloadDir}${CONST.RECEIPTS_UPLOAD_PATH}`; (_Download_).
- IOS: `${RNFetchBlob.fs.dirs.DocumentDir}${CONST.RECEIPTS_UPLOAD_PATH}`; (_Document_).